core/src/test/scala/org/apache/spark/sql/delta/expressions/InterleaveBitsSuite.scala

```diff
@@ -98,4 +103,26 @@ class InterleaveBitsSuite extends SparkFunSuite with ExpressionEvalHelper {
     InterleaveBits(Seq(Literal(0.toDouble))).checkInputDataTypes() != TypeCheckSuccess
     InterleaveBits(Seq(Literal(0.toString))).checkInputDataTypes() != TypeCheckSuccess
   }
+
+  test("randomization interleave bits") {
```

Comment:
how long does this test take with default 100mil rows?

Reply:
logs from GA
```
[info] InterleaveBitsSuite:
[info] - 0 inputs (85 milliseconds)
[info] - 1 input (304 milliseconds)
[info] - 2 inputs (26 milliseconds)
[info] - 3 inputs (30 milliseconds)
[info] - nulls (221 milliseconds)
[info] - consistency (253 milliseconds)
[info] - supported types (11 milliseconds)
[info] - randomization interleave bits (2 minutes, 29 seconds)
```

Should we set a smaller value?